framework/src/task_manager/task_manager_core.c

```diff
@@ -235,6 +238,10 @@ static int taskmgr_stop(int handle, int caller_pid)
 		return TM_NO_PERMISSION;
 	}
 
+	/* Call cancel callback */
+	tm_cancel_flag = TM_IS_CANCELLED;
```

Comment:
The name of flag looks like strange to me.
TM_IS_CANCELLING is better for this context.